.codeclimate.yml

```diff
@@ -0,0 +1,31 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
```

Comment:
Suspect it's just leaving on the ones that are there by default. If those languages exist in the project, they would be checked by that engine. Available engines:
https://docs.codeclimate.com/docs/list-of-engines